packages/stryker/test/unit/InputFileResolverSpec.ts

```diff
@@ -244,4 +246,22 @@ describe('InputFileResolver', () => {
         .throws('Cannot mutate web url "http://www".');
     });
   });
+
+
+  it(`should presume that files with an extension like .jpeg or .gif are binary`, async () => {
+    // Arrange
+    const binaryFiles = ['file.png', 'file.jpeg', 'file.jpg', 'file.gif', 'file.zip', 'file.tar'];
+    binaryFiles.forEach(file => globStub.withArgs(file).resolves([file]));
+    sut = new InputFileResolver([], binaryFiles, reporter);
+
+    // Act
+    const actual = await sut.resolve();
+
+    // Assert
+    expect(actual).deep.eq(binaryFiles.map(file => binaryFile({ 
+      name: path.resolve(file), 
+      mutated: false, 
+      transpiled: true 
```

Comment:
Why should `transpiled` be true?

Reply:
Because that is the default when no `transpiled` flag is given. At the same time the default for `binaryFile` is false.

Reply:
> At the same time the default for binaryFile is false.
But you just expect that it's true here.

Reply:
Yes it should be true. The `InputFileResolver` should set it to true by default, right?